python/pyhail/dataset.py

```diff
@@ -1644,21 +1986,227 @@ def variant_qc(self):
         pargs = ['variantqc']
         return self.hc.run_command(self, pargs)
 
-    def vep(self, config, block_size=None, root=None, force=False, csq=False):
+    def vep(self, config, block_size=1000, root="va.vep", force=False, csq=False):
```

Comment:
`'va.vep'`.  Single quotes for code and code-like strings, double-quotes "" for human readable things (error messages, documentation, etc.)